experimental/adaptive-tool/README.md

```diff
@@ -1,74 +1,110 @@
 # Bot Framework Adaptive Tool
-Bot Framework Adaptive Tool is a Visual Studio Code extension that helps developers handle LG (.lg), LU (.lu) files and Dialog (.dialog) files efficiently.
 
-The support includes:
+Bot Framework Adaptive Tool is a VS Code extension that helps developers handle LG (.lg), LU (.lu), and Dialog (.dialog) files efficiently. Adaptive Tool has tools and settings that make it easy to debug, analyze and enhance you language files.
```

Comment:
>LU (.lu) [](start = 90, length = 8)

Doesn't it work on .qna as well?  (Which is basically lu format.)

Reply:
No, only .lu is currently supported:
![qna-lu-adaptive-tool](https://user-images.githubusercontent.com/9682548/96307883-2a6ab480-0fb7-11eb-9a05-9e1a675852d2.png)

Reply:
Did you try adding qna as an lg extension?

---
In reply to: [506717510](https://github.com/microsoft/BotBuilder-Samples/pull/2819#discussion_r506717510) [](ancestors = 506717510)

Reply:
@chrimc62 No, I didn't. I just followed the instructions in the original README (install the extension, then open an lg, lu, or dialog file).